CONTRIBUTING.md

```diff
@@ -13,6 +13,17 @@ on how to become a [**Member**](https://github.com/open-telemetry/community/blob
 [**Approver**](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
 and [**Maintainer**](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer).
 
+## Find an OTel Buddy and Get Started Quickly!
```

Comment:
```suggestion
## Find a Buddy and Get Started Quickly!
```

Reply:
tongue in cheek trying to get a brand on OpenTelemetry buddies :) but I'll fix it up.